core/utils/utils.py

```diff
@@ -16,7 +16,7 @@
 def _rdf_base(request):
     host = request.get_host()
```

Comment:
Is this where the `host` variable that you've been replacing with `settings.BASE_URL` comes from?  If so, is anything else using it, or can it be done away with entirely?